util/options.hh

```diff
@@ -548,7 +548,7 @@ struct text_options_t : option_group_t
 		   "Only one of text and text-file can be set");
   }
 
-  const char *get_line (unsigned int *len);
+  const char *get_line (unsigned int *len, char eol);
```

Comment:
Maybe add the default `\n` here?

Reply:
Did it myself. Thanks agian.

Reply:
I considered that, but found it is called only in one place. But thinking again, I agree, it is more elegant/self documenting.